packages/html/ds/src/pagination/pagination.html

```diff
@@ -0,0 +1,158 @@
+{% from 'button/button.html' import govieButton %}
+
+{% macro goviePagination(props) %}
+  <div
+    class="gi-pagination"
+    data-current-page="{{ props.currentPage }}"
+    data-total-pages="{{ props.totalPages }}"
+    data-testid="govie-pagination"
+    data-module="gieds-pagination"
+  >
+    {{
+      govieButton({
+        "content": "<span data-testid=\"govie-icon\" role=\"presentation\" class=\"material-symbols-outlined gi-block gi-text-[24px]\">arrow_left_alt</span> <span class='sm:gi-block gi-hidden'>Previous</span>",
```

Comment:
I'm not sure if this will render an error but can we use the `govieIcon` macro here ? Otherwise if we change something for this component, the changes will not reflect here as we manually add the rendered html